util_scripts/Populate_Metadata.py

```diff
@@ -34,17 +34,23 @@
 from omero.util.populate_metadata import ParsingContext
 
 
-def get_original_file(conn, plateId, fileId):
-    plate = conn.getObject("Plate", int(plateId))
-    if plate is None:
-        sys.stderr.write("Error: Object does not exist.\n")
-        sys.exit(1)
+def get_original_file(conn, object_type, plate_id, file_id):
```

Comment:
The only thing here, and this is nitpicking, is that this variable should probably be called `object_id` rather than `plate_id` as it could be a Screen or Plate ID.